tests/sessions_tests/tests.py

```diff
@@ -708,14 +708,11 @@ def test_session_update_error_redirect(self):
         request.session.save(must_create=True)
         request.session.delete()
 
-        # Handle the response through the middleware. It will try to save the
-        # deleted session which will cause an UpdateError that's caught and
-        # results in a redirect to the original page.
-        response = middleware.process_response(request, response)
-
-        # Check that the response is a redirect.
-        self.assertEqual(response.status_code, 302)
-        self.assertEqual(response['Location'], path)
+        with self.assertRaises(SuspiciousOperation):
```

Comment:
We typically use `assertRaisesMessage` to make it easier to track where the exception originates.